src/ExposureHistoryContext.tsx

```diff
@@ -17,23 +17,23 @@ import {
   ExposureCalendarOptions,
   ExposureInfo,
 } from "./exposureHistory"
+import { GetStoreData, SetStoreData } from "./utils/General"
```

Comment:
this looks a bit out of date, we updated the utils folder a bit.

Reply:
Just rebased, it is up to date now.